src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/OverlayList.js

```diff
@@ -108,9 +114,12 @@ export default class OverlayList extends React.PureComponent<Props> {
     handleBackropClick = this.requestClose;
 
     render() {
-        const dimensions = this.dimensions;
-        const style = dimensions ? OverlayListPositioner.dimensionsToStyle(dimensions) : {opacity: '0'};
-        this.scrollTop = dimensions ? dimensions.scrollTop : 0;
+        let style = {opacity: '0'};
```

Comment:
`Opacity: 0` is the only option. The overlay-list gets rendered two times when its opened. First with 0 opacity, then the dimensions are calculated and then its rendered again. Otherwise the overlay list would jump as it does in the current develop. Moreover I want to be able to focus the elements in the overlaylist which is just possible with 0 opacity and not with visibility hidden or display none. But if the select is closed the overlay list isn't even in the DOM. So nothing to worry about blocking clicks.